helm/Makefile

```diff
@@ -13,7 +14,7 @@ SUBCHARTS:=idol-library \
 
 PACKAGE_SUBCHARTS:=$(addprefix package-,${SUBCHARTS})
 CLEAN_SUBCHARTS:=$(addprefix clean-,${SUBCHARTS})
-README_SUBCHARTS:=$(addsuffix /README.md,idol-omnigroupserver)
+README_SUBCHARTS:=$(addsuffix /README.md,idol-omnigroupserver idol-community)
```

Comment:
Maybe could use `$(wildcard ...)` (or similar) to check for .gotmpl files to save maintaining a list here?